infer/src/java/jFrontend.ml

```diff
@@ -115,37 +115,17 @@ let add_cmethod source_file program icfg cm proc_name =
             L.result "@\n" )
 
 
-let path_of_cached_classname cn =
-  let root_path = ResultsDir.get_path JavaClassnamesCache in
```

Comment:
@jvillard this code is why the warnings were reported `infer-out` before. It is expected to have things reported on the libraries with the `--dependencies` option enabled. It will now create fake files of the form  `/my/package/Lib.class` when the class name is `my.package.Lib.class`. However, for the purpose if this test, we suppress the reports on the libraries to make the test more stable when the version of Java is changing.